deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -356,7 +358,9 @@ def __init__(self,
                     self.parallel_partitioned_bit16_groups[i][partition_id].to(
                         self.device).clone().half().detach())
 
-            # modify optimizer of have flat master weight
+            # modify optimizer of have flat master weight.
```

Comment:
Thanks for this documentation improvements. However, the first line here seems to have a typo. Can you please fix? Thanks!

Reply:
Ok, I've re-commented this.

Reply:
> Thanks for this documentation improvements. However, the first line here seems to have a typo. Can you please fix? Thanks!

Ok, I've re-commented this.